docs/source/deployment/now.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Deploying with Zeit Now
+sidebar_title: Now
+description: Deploying your GraphQL server to Zeit Now
+---
+
+`now` is a service by Zeit that allows you to deploy an instance of Apollo Server and have a functional GraphQL endpoint. `now` supports three types of deployments for web apps: `static`, `node.js`, and `Docker`. We'll focus on the Node.js deployment for this guide.
```

Comment:
I think we can just call it "Now" (upper-case, as they refer to it on their site when they aren't directly talking about the `now` tool), and I don't think we need to mention the static site and Docker options at all (nor do I understand the significance of putting "node.js" and "Docker" in `code` format)